### PR TITLE
add Community helps to getting started page and add 2 solid apps to apps page

### PR DESCRIPTION
**Pull request by soltanireza65**

### CODE REVIEW

File: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md [hunk 1]
```diff
@@ -34,3 +34,19 @@ to run on your own machine.
 ## Developing your first app
 Once you are set up with your own Pod and WebID,
 you're ready to start [developing your own app](/developers/tutorials/first-app).
+
+## Community helps
```
Comment (TallTed) [inline _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:38]:
```suggestion
## Community assistance
```

File: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md [hunk 2]
```diff
@@ -34,3 +34,19 @@ to run on your own machine.
...
+
+Check some of the applications. Some interesting approaches
```
Comment (TallTed) [inline _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:40]:
```suggestion
Check some of the applications. Some interesting approaches follow:
```

File: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md [hunk 3]
```diff
@@ -34,3 +34,19 @@ to run on your own machine.
...
+
+- PodOS https://github.com/pod-os/PodOS (Web Components)
+- solid-uix https://github.com/SolidOS/solid-uix (Native Web Components)
```
Comment (TallTed) [inline _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:43]:
```suggestion
- **PodOS** (Web Components) https://github.com/pod-os/PodOS
- **solid-uix** (Native Web Components) https://github.com/SolidOS/solid-uix
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md (hunk 1 above)
Comment (soltanireza65) [reply _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md]: fixed

File: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md (hunk 2 above)
Comment (soltanireza65) [reply _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md]: appliyed

File: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md (hunk 3 above)
Comment (soltanireza65) [reply _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md]: appliyed

File: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md [hunk 4]
```diff
@@ -34,3 +34,19 @@ to run on your own machine.
...
+
+Solid With web frameworks.
+
+- ember-solid: [Video Tutorial](https://nextcloud.hellofuture.be/s/nYYNwGceFSNotJY)  which should be start-to-finish Solid Ember App (Aad Versteden).
+- take a look at [todo-ldo](https://github.com/megoth/todo-ldo) a [solid/nextjs/ts] app (Arne Hassel)
+- and also this one [solid-react-app-tutorial](https://virginiabalseiro.com/blog/solid-react-app-tutorial)
+
+Cou can also checkout the solid apps at:
+- [Solid Apps](https://solidproject.org/apps)
```
[the remark this answers is not in the document]
Comment (soltanireza65) [reply _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md]: appliyed

**Comment by soltanireza65** [comment]
@TallTed 
I've just added a new section "Youtube videos"

File: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md [hunk 5]
```diff
@@ -36,20 +36,23 @@ For instructions on:
 ## Developing your first app
 
 Once you are set up with your own Pod and WebID,
-you're ready to start [developing your own app](/developers/tutorials/first-app).
+you're ready to develop your own app. Various Solid client libraries may provide a beginning app tutorial; for example, the [Inrupt JS Client Library tutorial](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/getting-started/).
```
Comment (TallTed) [inline _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:39]:
```suggestion
you're ready to develop your own app. Various Solid client libraries may provide a  tutorial for beginners; for example, Inrupt offers the [JS Client Library tutorial](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/getting-started/).
```

File: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md [hunk 6]
```diff
@@ -36,20 +36,23 @@ For instructions on:
...
 
-## Community helps
+## Community assistance
 
-Check some of the applications. Some interesting approaches
+Check some of the applications. Some interesting approaches follow:
```
Comment (TallTed) [inline _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:43]:
```suggestion
Check out some of the existing applications. Some interesting approaches follow:
```

File: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md [hunk 7]
```diff
@@ -36,20 +36,23 @@ For instructions on:
...
+- **[solid-react-app-tutorial](https://virginiabalseiro.com/blog/solid-react-app-tutorial)**
 
-Cou can also checkout the solid apps at:
-- [Solid Apps](https://solidproject.org/apps)
\ No newline at end of file
+You can also look into the Solid apps at:
+- [Solid Apps](https://solidproject.org/apps)
+
+Youtube videos:
+- [How we're doing Solid - In less than 5 minutes](https://www.youtube.com/watch?v=CqI15zs1NHw)
```
Comment (TallTed) [inline _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:58]:
```suggestion
- [How we're doing Solid — in fewer than 5 minutes](https://www.youtube.com/watch?v=CqI15zs1NHw)
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Small stuff

**Comment by soltanireza65** [comment]
> Small stuff

fixed

**Comment by michielbdejong** [comment]
@TallTed can you re-review please?

**Review by TallTed — CHANGES_REQUESTED** [review]
Different small stuff :-)

File: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md [hunk 8]
```diff
@@ -36,4 +36,23 @@ For instructions on:
 ## Developing your first app
 
 Once you are set up with your own Pod and WebID,
-you're ready to develop your own app. Various Solid client libraries may provide a beginning app tutorial; for example, the [Inrupt JS Client Library tutorial](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/getting-started/).
+you're ready to develop your own app. Various Solid client libraries may provide a  tutorial for beginners; for example, Inrupt offers the [JS Client Library tutorial](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/getting-started/).
```
[the remark this answers is not in the document]
Comment (soltanireza65) [reply _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md]: applied

File: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md [hunk 9]
```diff
@@ -36,4 +36,23 @@ For instructions on:
...
+
+## Community assistance
+
+Check out some of the existing applications. Some interesting approaches follow:
+
+- **PodOS** (Web Components) https://github.com/pod-os/PodOS
+- **solid-uix** (Native Web Components) https://github.com/SolidOS/solid-uix
+
+Solid With web frameworks:
```
[the remark this answers is not in the document]
Comment (soltanireza65) [reply _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md]: applied

File: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md [hunk 10]
```diff
@@ -36,4 +36,23 @@ For instructions on:
...
+
+- **ember-solid**: [Video Tutorial](https://nextcloud.hellofuture.be/s/nYYNwGceFSNotJY)  which should be start-to-finish Solid Ember App (Aad Versteden)
```
[the remark this answers is not in the document]
Comment (soltanireza65) [reply _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md]: applied

File: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md [hunk 11]
```diff
@@ -36,4 +36,23 @@ For instructions on:
...
+- **[todo-ldo](https://github.com/megoth/todo-ldo)** a [solid/nextjs/ts] app (Arne Hassel)
+- **[solid-react-app-tutorial](https://virginiabalseiro.com/blog/solid-react-app-tutorial)**
```
[the remark this answers is not in the document]
Comment (soltanireza65) [reply _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md]: applied

**Comment by soltanireza65** [comment]
@TallTed 
I've just applied the requested changes,
As part of the last 2 commits, I've just added my new 2 solid apps to the apps page.
that's why I've just updated the PR title.

**Review by michielbdejong — APPROVED** [review]

**Comment by TallTed** [comment]
@soltanireza65 — It is often better to use the "Commit suggestion" or "Add suggestion to batch" buttons, as these both apply the suggested change *and* resolve the conversation without requiring manual review of manually applied changes (which I'm doing now)....

**Review by TallTed — APPROVED** [review]

**Pull request merged**